frontend/public/components/utils/resource-log.tsx

```diff
@@ -243,148 +271,358 @@ export const LogControls: React.FC<LogControlsProps> = ({
   };
   const label = t('public~Debug container');
   return (
-    <div className="co-toolbar">
-      <div className="co-toolbar__group co-toolbar__group--left">
-        <div className="co-toolbar__item">{showStatus()}</div>
-        {dropdown && <div className="co-toolbar__item">{dropdown}</div>}
-        <div className="co-toolbar__item">{logTypeSelect(!hasPreviousLog)}</div>
-        <div className="co-toolbar__item">
-          <LogViewerSearch
-            onFocus={() => {
-              if (status === STREAM_ACTIVE) {
-                toggleStreaming();
-              }
-            }}
-            placeholder="Search"
-            minSearchChars={0}
-          />
-        </div>
-        {showDebugAction(resource, containerName) && !isWindowsPod(resource) && (
-          <Link
-            to={`${resourcePath(
-              'Pod',
-              resource.metadata.name,
-              resource.metadata.namespace,
-            )}/containers/${containerName}/debug`}
-            data-test="debug-container-link"
-          >
-            {label}
-          </Link>
-        )}
-        {showDebugAction(resource, containerName) && isWindowsPod(resource) && (
-          <Tooltip
-            content={t(
-              'public~Debug in terminal is not currently available for windows containers.',
+    <Toolbar>
+      <ToolbarContent visibility={{ default: isMobile ? 'visible' : 'hidden' }}>
```

Comment:
I missed this the first time.  Since you're using the `isMobile` hook, I would not use `visibility` but rather change the output.  e.g., 
```
{ isMobile && (
  <ToolbarContent>
  ...
  </ToolbarContent>
))

Reply:
Because what's being output to the DOM changes rather than just the visibility (CSS).